### PR TITLE
ANC - Remove the instruction Text on Symptom review activity

### PR DESCRIPTION
**Pull request by Diama1**
[#325 Comment](https://github.com/TIP-Global-Health/eheza-app/issues/325#issuecomment-1191534837)

### CODE REVIEW

**Comment by Diama1** [comment]
Screenshot from 2022-08-03 15-02-32

File: client/src/elm/Pages/Prenatal/Activity/View.elm [hunk 1]
```diff
@@ -2101,9 +2101,7 @@ viewSymptomReviewContent language currentDate assembled data =
     , div [ class "ui full segment" ]
         [ div [ class "full content" ]
             [ div [ class "ui form symptom-review" ] <|
-                div [ class "instructions" ]
```
Comment (anvmn) [inline client/src/elm/Pages/Prenatal/Activity/View.elm:2061]: @Diama1 This removes the label from both screens of activty.
It should be removed only from first screen.

Peek 2022-08-07 12-18

**Review by anvmn — CHANGES_REQUESTED** [review]

**Comment by Diama1** [comment]
imagepp

File: client/src/elm/Pages/Prenatal/Activity/View.elm [hunk 2]
```diff
@@ -2053,15 +2053,25 @@ viewSymptomReviewContent language currentDate assembled data =
...
+
+        instructionLabel =
+            case data.step of
+                SymptomReviewStepSymptoms ->
+                    [ div [ class "ui form symptom-review" ] <|
+                        inputs
+                    ]
+
+                SymptomReviewStepQuestions ->
+                    [ div [ class "ui form symptom-review" ] <|
```
Comment (anvmn) [inline client/src/elm/Pages/Prenatal/Activity/View.elm]: Code duplication.

**Review by anvmn — CHANGES_REQUESTED** [review]
@Diama1 

Condition shoud be if to show or not:

```
div [ class "instructions" ]
     [ viewLabel language Translate.PrenatalSymptomQuestionsHeader ]
```

**Review by anvmn — APPROVED** [review]
:+1:

**Pull request merged**